beacon/app/beacon.hs

```diff
@@ -498,6 +508,24 @@ parseOpts =   BenchmarksCompareOptions
                       ]
                     )
 
+    parseConfigPath :: Parser FilePath
+    parseConfigPath = strOption
+                      ( mconcat
+                        [ long "config-path"
+                        , value "/configuration/cardano/mainnet-config.json"
+                        , help "File path to the config.json file. Relative to node-home. When not passed this defaults to /configuration/cardano/mainnet-config.json"
+                        ]
```

Comment:
We could add a local `default` definition so that we don't have to change the default value in two places. It seems less error-prone, but the duplication is quite localized, so feel free to leave it as is.

Reply:
Even if what we have is

```
[ combinator "data", combinator "data",...]
```